tools/topology/m4/pipeline.m4

```diff
@@ -7,6 +7,13 @@ define(`SCHEDULE_TIME_DOMAIN_TIMER', 1)
 # default number of DAI periods
 define(`DAI_DEFAULT_PERIODS', 2)
 
+# default stereo channel map
+ifdef(`VOL_CH_MAPS', , `define(`VOL_CH_MAPS', LIST(`	', KCONTROL_CHANNEL(FL, 1, 0),
+							   KCONTROL_CHANNEL(FR, 1, 1)))')
+
+ifdef(`SW_CH_MAPS', , `define(`SW_CH_MAPS', LIST(`	', KCONTROL_CHANNEL(FL, 2, 0),
+							   KCONTROL_CHANNEL(FR, 2, 1)))')
+
```

Comment:
what does "SW" stand for? Software?  Switch?

Reply:
yes SWITCH... I can rename it with longer and more clear name